sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/src/Generated/Models/Role.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.DeviceUpdate.Models
+{
+    /// <summary> Whether the location is primary or failover. </summary>
+    public readonly partial struct Role : IEquatable<Role>
```

Comment:
Same as above, please rename this one.

Reply:
Same question, are we allowed to rename generated files?

Reply:
Same as above.